src/Facebook/InstantArticles/Transformer/Getters/JSONGetter.php

```diff
@@ -10,45 +10,27 @@
 
 use Facebook\InstantArticles\Validators\Type;
 
-class JSONGetter extends ChildrenGetter
+class JSONGetter extends AbstractGetter
 {
-    /**
-     * @var string
-     */
-    protected $attribute;
-
-    public function createFrom($properties)
+    public function createFrom(array<string, string> $properties): this
     {
-        if (isset($properties['selector'])) {
+        if (array_key_exists('selector', $properties)) {
             $this->withSelector($properties['selector']);
         }
-        if (isset($properties['attribute'])) {
+        if (array_key_exists('attribute', $properties)) {
             $this->withAttribute($properties['attribute']);
         }
-    }
-
-    /**
-     * @param string $attribute
-     *
-     * @return $this
-     */
-    public function withAttribute($attribute)
```

Comment:
Has this moved somewhere else?

Reply:
This method already exists on `AbstractGetter`, so after the refactor you can remove it.

Reply:
As @diegoquinteiro says.
Removed for no needing to keep it here.